plutus-tx/src/PlutusTx/Builtins/Internal.hs

```diff
@@ -408,6 +409,10 @@ chooseList :: BuiltinList a -> b -> b -> b
 chooseList (BuiltinList [])    b1 _ = b1
 chooseList (BuiltinList (_:_)) _ b2 = b2
 
+{-# OPAQUE drop #-}
+drop :: Integer -> BuiltinList a -> BuiltinList a
```

Comment:
Why not `dropList`?

Reply:
For consistency, we also have `head` (and not `headList`) and `tail` (and not `tailList`) in this file.